server/src/main/java/info/doseamigos/meds/Med.java

```diff
@@ -0,0 +1,93 @@
+package info.doseamigos.meds;
+
+import info.doseamigos.amigousers.AmigoUser;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * POJO for a medicine.  This is what we store for each user who is taking medicine.
+ * Note that rxcui comes from an external rest service when we do lookups.  If one cannot
+ * be found, rxcui can be null.
+ */
+public class Med {
+    private long medId;
+    private AmigoUser user;
+    private Long rxcui;
```

Comment:
The reason this is Long and not long is because I think it should be nullable.  Consider the situation that some medicine a user is taking isn't in the external service.  I would just leave this null, and we wouldn't be able to find any drug interaction information for this specific Med.  Unit tests reflect that this is nullable.